src/Surfnet/ServiceProviderDashboard/Domain/Entity/ManageEntity.php

```diff
@@ -132,6 +132,15 @@ public function getMetaData(): ?MetaData
         return $this->metaData;
     }
 
+    public function updateStatusByExcludeFromPush(): void
+    {
+        if ($this->isExcludedFromPush()) {
+            $this->updateStatus(Constants::STATE_PUBLICATION_REQUESTED);
+            return;
+        }
+        $this->updateStatus(Constants::STATE_PUBLISHED);
```

Comment:
Behavior-wise your change is slightly different from the original. And in this case, the original (wonky looking) behavior is actually intentional.

The coin value for exclude from push can have three different values. Either 1, 0 or null. Where null means it is not set. And 1, that is should be excluded, and 0 that it should not.

When the coin value is `null`, the status should not be updated.